### ISSUE TITLE
Public documentation of block/suspension changes?

### ISSUE THREAD

**Issue by sindarina** [hachyderm/community#175]
[Yesterday's blog post](https://community.hachyderm.io/blog/2022/12/05/updating-domain-blocks/) mentions another thirteen (13) domains being removed from the Hachyderm suspension list, but, as far as I can tell, there is no longer a publicly maintained blocklist here on Github, as mentioned in [Quintessence's Mastodon reply on the topic](https://hachyderm.io/@quintessence/109389031485272331). The [linked repository](https://github.com/hachyderm/hack) mentions being archived.

Is there a new location for this data, or another easy way to track which domains are being added and/or removed? Managing said list via the Mastodon tooling makes total sense, but it would still be nice to have a timestamped record of those actions/decisions, and a bit more detail than "we removed 'domain.example' from the list, plus 13 others", so we don't have to manually check the list ourselves if we want to know?

Thanks!

**Comment by quintessence** [comment]
👋 Mod here

The current place to view our moderated domains is our About page:

https://hachyderm.io/about

This list includes both "limited" and "blocked" domains. The `hack` repo was d/c'ed because we noticed it was falling out of sync with the source of truth, and the list generated on the About page is generated by the Mastodon tooling based on what's currently applied.

**Comment by clintharrison** [comment]
@quintessence does the "About" page provide any way to see how the list changes over time? That was simple to do when it was in a Git repo.

Can you say more about choosing to discontinue the repo rather than better integrating it into the process of blocking a new instance?

**Comment by quintessence** [comment]
I will take time to add a page to our docs about Blocklists to help answer commonly asked questions.

The short version is that the `hack` repo was one way of many for us to bridge gaps in the tooling. When we changed processes, we no longer had that need.

If you have more questions about Mastodon please feel free to open up a new issue! I'm going to lock this one mainly because it's hard to follow up on comments added to issues once they're closed, but new issues for questions / concerns are welcome :)
